### ISSUE TITLE
double start() doubles prefix and suffix

### ISSUE THREAD

**Issue by JimWinters** [WoLpH/python-progressbar#229]
#### Description

I was getting my suffix twice.  Some considerable wondering and debugging eventually revealed that I was calling start() on a progress bar twice before actually using the progress bar.  (I didn't notice, because the calls were in separate sections of code.)  Removing the extra start() solved my problem.

It is unclear to me whether it might be desirable to call start() multiple times in some applications.  For my application, it is not desirable, so this was a bug in my code.  In any case, doubling the prefix and suffix is probably not the intended outcome.  :-)

#### Code

```python
#!/usr/bin/python3

import progressbar
import sys
import time

def basic_widget_example():
    prefix = "beg "
    suffix = " end"
    widgets = [progressbar.Percentage()]
    bar = progressbar.ProgressBar(widgets=widgets, max_value=10,
                                  prefix=prefix, suffix=suffix).start()
    bar.start()                 # My bug.
    for i in range(10):
        # do something
        time.sleep(0.1)
        bar.update(i + 1)
    bar.finish()

if __name__ == '__main__':
    basic_widget_example()
```

#### Versions

- Python version:

```
 $ python3
Python 3.5.2 (default, Apr 16 2020, 17:47:17) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys; print(sys.version)
3.5.2 (default, Apr 16 2020, 17:47:17) 
[GCC 5.4.0 20160609]
```

- Python distribution/environment: CPython

- Operating System: Linux Mint 18.3

- Package version: 3.51.1

**Comment by WoLpH** [comment]
That's indeed a bug. It's now fixed in 3.51.2 :)

**Comment by JimWinters** [comment]
I see it works.  Thank you.
